src/analysis/normed_space/multilinear.lean

```diff
@@ -401,6 +421,23 @@ def prodL :
   right_inv := λ f, by ext; refl,
   norm_map' := λ f, op_norm_prod f.1 f.2 }
 
+/-- `continuous_multilinear_map.pi` as a `linear_isometry_equiv`. -/
+def piₗᵢ {ι' : Type v'} [fintype ι'] {E' : ι' → Type wE'} [Π i', normed_group (E' i')]
+  [Π i', normed_space 𝕜 (E' i')] :
+  @linear_isometry_equiv 𝕜 (Π i', continuous_multilinear_map 𝕜 E (E' i'))
+    (continuous_multilinear_map 𝕜 E (Π i, E' i)) _ _ _
+      (@pi.semimodule ι' _ 𝕜 _ _ (λ i', infer_instance)) _ :=
```

Comment:
Is there really no way to have Lean find this instance by itself?

Reply:
I tried and failed.